### ISSUE TITLE
Project fork needed

### ISSUE THREAD

**Issue by enkessler** [funkwerk/gherkin_lint#23]
Because

1. There is community support to continue developing this project
2. The only person with merge/release/etc. permission (i.e. @lindt ) is no longer available

there is a need to fork this project.

So, @johngluckmdsol , @mnohai-mdsol, and anyone else with an opinion: where will the fork be?

**Comment by mjnohai** [comment]
I posted an issue on one of his most recent projects in hope he can give us contributor access to continue the project. Lets see

**Comment by enkessler** [comment]
Oh, and there is also a third option: just make a brand new gem that we can publish/maintain ourselves.

**Comment by johngluckmdsol** [comment]
I assume there's potential licensing/IP issues with that

**Comment by enkessler** [comment]
No, not really. What kind of issues are you thinking about?

**Comment by enkessler** [comment]
I'll just leave this here...

https://github.com/enkessler/cuke_linter

**Comment by mjnohai** [comment]
Awesome!!! I'm sure we can copy the code over and site original author and all that jazz we'd be good to go.

**Comment by enkessler** [comment]
Inasmuch as it looks anything like the current code base, sure, I suppose.

Honestly I was just going to write it from scratch. Partly for the fun of it and partly because the internal approach to running all of the linters won't look the same because it will be using `cuke_modeler`. Remember, I was going out of my way in #20 to change as little as possible. Now that it's a new project, we might as well write it how we like right from the get go.

**Comment by johngluckmdsol** [comment]
Yeah, I have no problem with rewriting it.  I think `rubocop` has the right idea as far as linters go and I'd support the project adopting the same structure.  I'm happy to try to contribute but I don't have a ton of spare time outside of work.  I'm not entirely happy with the approach existing in this project.  I was planning on doing some refactoring when I got time and if the need hit. Our changes were starting to move the project more toward the `rubocop` way.

**Comment by enkessler** [comment]
It's been quiet and slow but I finally have the first release out.

https://rubygems.org/gems/cuke_linter
